package.json

```diff
@@ -8,6 +8,7 @@
     "start": "ts-node src/index.ts",
     "test": "jest",
     "build": "tsc",
+    "postinstall": "npm i && tsc",
```

Comment:
Afaik, the `postinstall` hook is run after running `npm i`, so adding it to the command seems redundant. What made you add this to the hook?